<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <widget class="QLineEdit" name="lineEditname">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>160</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditpwd">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>230</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="loginbtn">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>340</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="registerbtn">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>340</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelname">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>161</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelpwd">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>231</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelmsg">
       <property name="geometry">
        <rect>
         <x>203</x>
         <y>289</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="registerpage">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TCP address</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>key</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="seraddlineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="seruserlineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="serpwdlineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>200</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="registerbtn1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>360</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cleanbtn">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>360</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>登陆用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>登陆密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="register_username">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="register_pwd">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>320</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="backbtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pagemid"/>
     <widget class="QWidget" name="page2">
      <widget class="QPushButton" name="ledbtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>开</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>灯光：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ledclosebtn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>关</string>
       </property>
      </widget>
      <widget class="QPushButton" name="air_open_btn">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>开</string>
       </property>
      </widget>
      <widget class="QPushButton" name="air_close_btn">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>关</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>   空调：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addbtn">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="jianbtn">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
